src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -189,5 +189,34 @@ void conjured_update_qualityDropsTwiceAsFastAsNormalItemsWhenSellInIsPassed0() {
         assertEquals(28, app.items[0].quality);
     }
 
+    @Test
+    void redWine_update_qualityDropsThreeTimesSlowerThanNormalItems() {
+        String name = "Red Wine";
+        int sellInDays = 7;
+        int quality = 32;
+        Item[] items = new Item[] { new Item(name, sellInDays, quality) };
+        GildedRose app = new GildedRose(items);
+
+        app.updateQuality();
+        app.updateQuality();
+        app.updateQuality();
+
+        assertEquals(31, app.items[0].quality);
```

Comment:
**Suggestion**: Possible to attach this number to a named variable for context, instead of having it be a magic number?

Reply:
Addressed in [this commit](https://github.com/jramnani/GildedRoseJavaLLC/pull/19/commits/b07ad63df52e51928c4edde22a1e2c08413ada37)